### ISSUE TITLE
Git submodule command fails on Windows

### ISSUE THREAD

**Issue by mattvenz** [buildkite/agent#1058]
This command works on Linux but fails on Windows. 
```
> git submodule foreach --recursive git reset --hard
--
  | error: unknown option `hard'
```
Wrapping the nested command in quotes might fix the problem.
```
> git submodule foreach --recursive "git reset --hard"
```
https://github.com/buildkite/agent/blob/9051b0dd535a14d2bd5bd9a51c1239b9e5a2ceda/bootstrap/bootstrap.go#L1203

**Comment by lox** [comment]
Sorry you ran into this, it was fixed in https://github.com/buildkite/agent/pull/1029 and released in v3.13.2!

**Comment by mattvenz** [comment]
> Sorry you ran into this, it was fixed in #1029 and released in v3.13.2!

Thanks!
